staging/src/k8s.io/sample-controller/README.md

```diff
@@ -40,7 +40,7 @@ This is an example of how to build a kube-like controller with a single type.
 
 ```sh
 # assumes you have a working kubeconfig, not required if operating in-cluster
-$ go run *.go -kubeconfig=$HOME/.kube/config
+$ go build && ./sample-controller -kubeconfig=$HOME/.kube/config
```

Comment:
@jotacamou Thanks for your PR and sorry that it took so long to get a review.

The error is now fixed and this update won't be needed.
/close